_assets/javascripts/views/live-stream.js

```diff
@@ -14,91 +14,83 @@ function liveStreamInit() {
   //   checkOrigin: false
   // }, '#giveIframe');
 
-  var dontMissCards = document.getElementsByClassName('.carousel--dont-miss .card');
+  var dontMissCards = document.getElementsByClassName(
+    ".carousel--dont-miss .card"
+  );
   for (var i = 0; i < dontMissCards.length; i += 1) {
-    dontMissCards[i].classList.add('carousel-cell');
+    dontMissCards[i].classList.add("carousel-cell");
   }
 
   // Buttons for Card Carousel
-  $('.btn.carousel-prev').on('click', function () {
-    var id = $(this).closest('.well').find('.card-deck.carousel').data('carousel-id');
+  $(".btn.carousel-prev").on("click", function() {
+    var id = $(this)
+      .closest(".well")
+      .find(".card-deck.carousel")
+      .data("carousel-id");
     var carousel = CRDS._instances[id];
     carousel.flickity.previous();
-  })
-  $('.btn.carousel-next').on('click', function () {
-    var id = $(this).closest('.well').find('.card-deck.carousel').data('carousel-id');
+  });
+  $(".btn.carousel-next").on("click", function() {
+    var id = $(this)
+      .closest(".well")
+      .find(".card-deck.carousel")
+      .data("carousel-id");
     var carousel = CRDS._instances[id];
     carousel.flickity.next();
-  })
+  });
 
   var search = window.location.search.substring(1);
   var debug = false;
   if (search) {
-    var params = JSON.parse('{"' + decodeURI(search).replace(/"/g, '\\"').replace(/&/g, '","').replace(/=/g,
-      '":"') + '"}');
-    debug = (params.debug == 'true');
+    var params = JSON.parse(
+      '{"' +
+        decodeURI(search)
+          .replace(/"/g, '\\"')
+          .replace(/&/g, '","')
+          .replace(/=/g, '":"') +
+        '"}'
+    );
+    debug = params.debug == "true";
   }
 
-  CRDS.Countdown.isStreamLive().then(function (isLive) {
+  CRDS.Countdown.isStreamLive().then(function(isLive) {
     if (!isLive && !debug) {
-      window.location.href = '/live';
+      window.location.href = "/live";
     }
   });
 
   // Analytics for Roll Call
   var successInterval;
-  document.addEventListener("DOMContentLoaded", function () {
-    var el = document.querySelector('roll-call');
+  document.addEventListener("DOMContentLoaded", function() {
+    var el = document.querySelector("roll-call");
 
     // These events are deprecated.
-    el.addEventListener('success', function (event) {
-      successInterval = window.setInterval(trackLocation.bind(null, event), 100);
-    });
-    el.addEventListener('dismissed', function (event) {
-      analytics.track('RollCallDismissed', {
-        Source: 'Crossroads.net'
+
+    el.addEventListener("dismissed", function(event) {
+      analytics.track("RollCallDismissed", {
```

Comment:
This event was missed in the original refinement of the Card. Could we also remove the "RollCallDismissed" event?